keyboards/karn/keymaps/colemak/keymap.c

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Robert Mills (@robcmills)
+// SPDX-License-Identifier: GPL-2.0
+
+#include QMK_KEYBOARD_H
+
+// Tap Dance declarations
+enum {
+    TD_F1_F11,
+    TD_F2_F12,
+};
+
+// Tap Dance definitions
+qk_tap_dance_action_t tap_dance_actions[] = {
+    // Tap once for F1, twice for F11
+    [TD_F1_F11] = ACTION_TAP_DANCE_DOUBLE(KC_F1, KC_F11),
+    [TD_F2_F12] = ACTION_TAP_DANCE_DOUBLE(KC_F2, KC_F12),
+};
+
+// Equivalent to zmk behavior-hold-tap hold-preferred flavor
+#define HOLD_ON_OTHER_KEY_PRESS_PER_KEY
+
```

Comment:
```suggestion
```

Reply:
[karn 1b888a811b] Remove #define left in colemak/keymap.c